### PR TITLE
Feature/drive distance command

### PR DESCRIPTION
**Pull request by johnli01**

### CODE REVIEW

File: Competition2018/src/main/java/competition/subsystems/drive/commands/DriveForDistanceCommand.java [hunk 1]
```diff
@@ -101,7 +101,7 @@ public boolean isFinished() {
     @Override
     public void end() {
         log.info("Ending, PreviousPosition was " + startingPosition + ", Targeted Delta in distance is " + deltaDistance
-                + " Distance traveled is " + (deltaDistance - startingPosition));
```
Comment (JohnGilb) [inline Competition2018/src/main/java/competition/subsystems/drive/commands/DriveForDistanceCommand.java:104]: `", Current position should be " + (deltaDistance + startingPosition));`

**Review by JohnGilb — CHANGES_REQUESTED** [review]

File: Competition2018/src/main/java/competition/subsystems/drive/commands/DriveForDistanceCommand.java (hunk 1 above)
Comment (WasabiFan) [reply Competition2018/src/main/java/competition/subsystems/drive/commands/DriveForDistanceCommand.java:104]: Why? I would prefer having it print the target distance and the actual distance, as it was previously. Deltas are easier to compare than absolutes.

Comment (JohnGilb) [reply Competition2018/src/main/java/competition/subsystems/drive/commands/DriveForDistanceCommand.java:104]: @WasabiFan 

With the current changes, it's printing:
* Starting position
* Delta distance
* Final position

Before, it was printing:
* Starting position
* Delta distance
* the difference between delta distance and final position (a meaningless number?)

Comment (WasabiFan) [reply Competition2018/src/main/java/competition/subsystems/drive/commands/DriveForDistanceCommand.java:104]: Ah, got it; this is the same math that was in the original PR. In that case, the preferred calculation would be `current - initial`.

**Review by JohnGilb — APPROVED** [review]

**Pull request merged**